parser/query_test.go

```diff
@@ -10,8 +12,16 @@ import (
 func TestQueryDocument(t *testing.T) {
 	testrunner.Test(t, "query_test.yml", func(t *testing.T, input string) testrunner.Spec {
 		doc, err := ParseQuery(&ast.Source{Input: input, Name: "spec"})
+
+		log.Printf("got error %#v", err)
+
+		var e *gqlerror.Error
```

Comment:
I think in a lot of these cases you can just ignore the conversion error and remove the conditional:
```go
var e, _ := err.(*gqlerror.Error)
```
if its not a gqlerror it will be nil, same effect, less code.

Reply:
I didn't think of that. Will fix that as soon as possible